File: BY-PYTHON/vowelorconsonent.py
def cheak_characters(char):
    vowel = "AEIOUaeiou"
    
    
    if char.isalpha():
        if char in vowel:
            print(f"{char} is a vowel character")
            
        else:
            print(f"{char} is a consonent character")
            
    else:
        print(f"{char} is not a character")
        
        
char = input("Enter a character: ")
x = cheak_characters(char)
print(x)

File: JAVA/VowelorConsonent.java
import  java.util.Scanner;


public class VowelorConsonent {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter   a character :");
        char  ch =  sc.next().charAt(0);



        if (Character.isLetter(ch)) {
            if ("AEIOUaeiou".indexOf(ch) != -1) {
                System.out.println(ch + " is a vowel.");
            
                
            }
            else {
                System.out.println(ch + " is a consonant.");
            }
            
        }

        else {
            System.out.println("Invalid input.");
        }
    }
}